test/fixtures/actual.js

```diff
@@ -27,9 +40,12 @@ class A {
         arg5: Array<string>,
         arg6: Class<SomeOtherClass>,
         arg7: Function,
-        arg7: Map<string, number>,
-        arg8: Set<string>,
-        arg9: Promise<string>
+        arg8: Map<string, number>,
+        arg9: Set<string>,
+        arg10: Promise<string>,
+        arg11?: int,

```

Comment:
Ok, thanks. Got it!
Didn't know about flowtype, so I have no idea what am I doing here! :wink: